ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -247,7 +248,7 @@ interface State {
   selectedCohortIds: number[];
   selectedConceptSetIds: number[];
   selectedPreviewDomain: string;
-  selectedValues: DomainValuePair[];
+  selectedValuePairs: DomainValuePair[];
```

Comment:
curious why you went with just `ValuePairs` in the front end and `DomainValuePairs` in the API.

Reply:
dunno - so the variable names wouldn't be overly long?  That's probably not a great reason.  I'll update to `selectedDomainValuePairs`

Reply:
I think thats a fair reason if you want to keep it as is.

Reply:
I don't think so BUT I am also a strong proponent for our naming being consistent between our code and our UI. Our UI currently calls these "Values" so I think our code should reflect that.

I do think there is a better name out there but I'd like product/design input and buy in.

Reply:
Yeah, my thought was that was non-ideal but unsafe to fix quickly